templates/bot/js/default/.env.teamsfx.local

```diff
@@ -0,0 +1,6 @@
+# TeamsFx will overwrite the following variable values when running debug
```

Comment:
Any plan to apply this change to scenario-based bot templates?

Reply:
Sure. Will update other templates as well before 10/21.